src/app/themes.tsx

```diff
@@ -116,6 +116,15 @@ const StyleManager: React.FC<Props> = ({ children, theme = lightTheme }) => {
         typography: {
             fontFamily: "Arial, Helvetica, sans-serif",
         },
+        components: {
+            MuiAppBar: {
```

Comment:
not sure if I'm a fan of manually setting colours via overriding exactly when you need each component - I would probably prefer the `themes.tsx` file to specify the palette of colours you _can_ use, and each component using them within styled-components - interested to what you think are the pros/cons of both approaches!